doc/man/man3/crypto_argon2i.3monocypher

```diff
@@ -172,7 +173,7 @@ const uint8_t  salt[16];                    /* random salt     */
 const uint32_t nb_blocks = 100000;          /* 100 megabytes   */
 const uint32_t nb_iterations = 3;           /* 3 iterations    */
 void *work_area = malloc(nb_blocks * 1024); /* work area       */
-if (work_area == 0) {
+if (work_area == NULL) {
```

Comment:
Good catch. I say `0` all over my code, but `NULL` is indeed better for the manual.